src/components/SearchResult/SearchResult.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import { Thumbnail } from '../..';
+import { SearchResult } from './SearchResult';
+import {
+	SearchResultSubtitle,
+	SearchResultThumbnail,
+	SearchResultTitle,
+} from './SearchResult.slots';
+import { fakeSearchResult } from './SearchResult.stories';
+
+const searchResult = (
+	<Router>
+		<SearchResult
+			type={fakeSearchResult.administrative_type as 'collection' | 'video' | 'audio'}
+			date={fakeSearchResult.dcterms_issued}
+			id={fakeSearchResult.pid}
+			description={fakeSearchResult.dcterms_abstract}
+			duration={fakeSearchResult.fragment_duration_time}
+			numberOfItems={25}
+			tags={['Redactiekeuze', 'Partner']}
+		>
+			<SearchResultTitle>
+				<Link to={`detail/${fakeSearchResult.pid}`}>{fakeSearchResult.dc_title}</Link>
+			</SearchResultTitle>
+			<SearchResultSubtitle>
+				<Link to={`search?filters={'provider':['${fakeSearchResult.original_cp}']}`}>
+					{fakeSearchResult.original_cp}
+				</Link>
+			</SearchResultSubtitle>
+			<SearchResultThumbnail>
+				<Link to={`detail/${fakeSearchResult.pid}`}>
+					<Thumbnail
+						category={fakeSearchResult.administrative_type as any}
+						src={fakeSearchResult.thumbnail_path}
+						label={fakeSearchResult.administrative_type}
+					/>
+				</Link>
+			</SearchResultThumbnail>
+		</SearchResult>
+	</Router>
+);
+
+describe('<SearchResult />', () => {
```

Comment:
Hier kunnen nog wat testen bij:

- it('Should be able to render a SearchResultTitle-slot'
- it('Should be able to render a SearchResultSubtitle-slot'
- it('Should be able to render a SearchResultThumbnail-slot'
- it('Should call onToggleBookmark when clicking the bookmark <ToggleButton />'
- it('Should call onToggleBookmark when clicking the bookmark <ToggleButton />'
- Een test voor elke prop om te zien of hij correct generenderd/doorgegeven/gebruikt wordt